backend/src/main/resources/db/migration/V8__insurance_number_unique.sql

```diff
@@ -0,0 +1,3 @@
+-- add insurance number unique constraint
+ALTER TABLE patients
+    ADD UNIQUE (insurance_number);
```

Comment:
to uz je ale mergnute, ne? Chyba githubu, predpokladam

Reply:
ano, to je divný, že to ukazuje. Přimergoval jsem teď master, takže už by to ukazovat neměl